### ISSUE TITLE
API uses deprecated host envvars

### ISSUE THREAD

**Issue by hedss** [balena-io/open-balena-api#48]
Most `HOST` variables have changed in name in the closed API, (eg. `VPN_HOST` is now `BALENA_VPN_HOST`, etc.). We should alter the open API to follow these changes.

**Comment by dfunckt** [comment]
@hedss did similar changes occur to VPN and registry?

BTW, to handle this, we'll need to:

1. update open-balena to pass both old and new env vars
2. update open-balena-api to use the env vars
3. update open-balena to remove old env vars

**Comment by hedss** [comment]
@dfunckt Yes, if you look at https://github.com/balena-io/open-balena-base/pull/62/files you'll see the list of altered HOST names. Each component was changed to rename them from RESIN_ to BALENA_ at the time.
